Small error message added when running changesets in a mono repo with… (#997)
changesets/changesets: .changeset/afraid-ways-buy.md

---
"@changesets/cli": patch
---

> Add error message when running changesets in a repo with pnpm/yarn workspaces configured by no packages yet

Reviewer: ```suggestion
Add error message when running changesets in a repo with workspaces configured but no packages yet
```
